lib/galaxy/config/__init__.py

```diff
@@ -836,6 +840,11 @@ def _process_config(self, kwargs: Dict[str, Any]) -> None:
         else:
             self.version_extra = extra_info
 
+        # Carbon emissions configuration
+        carbon_intensity_entry = get_carbon_intensity_entry(kwargs.get("geographical_server_location_code", ""))
+        self.carbon_intensity = carbon_intensity_entry["carbon_intensity"]
+        self.geographical_server_location_name = carbon_intensity_entry["location_name"]
```

Comment:
It seems that `carbon_intensity` and `geographical_server_location_name` are always computed here. Therefore I think they should not be part of the `config_schema.yml` since whatever value you set in the `galaxy.yml` config file will be ignored by this. Also, the fewer config options the better :)